src/status_im2/common/home/style.cljs

```diff
@@ -13,3 +13,28 @@
    :margin-right        6
    :weight              :semi-bold
    :size                :heading-1})
+
+(defn unread-indicator
+  [unread-count]
+  (let [right-offset (cond (> unread-count 99) -14
+                           (> unread-count 9)  -10
```

Comment:
I assume 99 is max-value? Maybe use a constant for `99` and `9` values like you declared in `src/quo2/components/counter/counter.cljs` to make more understandable.

Reply:
What do you think we should do @briansztamfater, because the thing is that the constant already exists, but I can only "talk" to the quo library via the `quo2.core` namespace.

Have you faced this problem before? Any preference on how the quo component should expose public properties?

Reply:
Hmmm good question, what do you think about moving the constant to `status-im2.constants` and require that namespace?

Reply:
That would work, but then the quo component would need to require a `status-im2` namespace, which I think we've been avoiding. Otherwise if I create another constant we will end up with two vars that should ideally be the same. See `max-value` is already defined in the quo component https://github.com/status-im/status-mobile/blob/c5da0c6423bd4a7b0249ca1465eb9cd9f0dadd90/src/quo2/components/counter/counter.cljs#L9

I don't know if it's any better, but we could have a `constants` namespace exposed from the `quo2` directory, so that we can reuse meaningful values on the outside. I honestly don't know if this is a good idea or not right now.

Reply:
Good point, thanks for pointing this out. I spend some time thinking about other solutions, and another approach would be making `max-value` a prop of the quo2 `counter` component, defining the constant in `status-im2.constants` and passing it through the props. In that way we make the `counter` component more configurable, we declare the constant in the constants namespace, and you can use it in this file. In my head sounds like the most clean solution, wdyt?

Reply:
That sounds really good @briansztamfater. I'll go for it!

Reply:
Done https://github.com/status-im/status-mobile/pull/15304/commits/f062f5707b98e563a040ec3f0012955946999c9d

I also updated the component tests to use the new test helpers and I added a test case for the configurable max value. As usual, thanks for the thoughtful feedback @briansztamfater.